docs/user-guide/recipes.rst

```diff
@@ -42,12 +42,13 @@ The pulp-docker plugin supports synchronizing from upstream repositories as of
 version 0.2.1. As of version 2.0.0, it can synchronize with either Docker v1 or
 v2 registries.
 
-.. note::
+.. warning::
+
+    ``index.docker.io`` is a legacy registry and should be used only for V1 content, along with
+    ``--enable-v1 true`` and ``--enable-v2 false``. Please note however that V1 content is
+    deprecated and Docker may remove it at any time.
+    For V2 content use ``registry-1.docker.io`` this is where the V2 registry API is located.
```

Comment:
Non-blocking suggestion:

s/For V2 content use ``registry-1.docker.io`` this is where the V2 registry API is located./
For V2 content use ``registry-1.docker.io``./